addons/web/static/src/core/commands/command_palette.scss

```diff
@@ -1,141 +1,30 @@
 .o_command_palette {
-  margin-top: 10vh;
+    $-app-icon-size: 1.8rem;
 
-  &.modal-content {
-    border: none;
-    border-radius: 3px !important;
-    overflow: hidden;
-  }
+    margin-top: 10vh;
```

Comment:
Could we use `.mt-5` instead?
Pros: 
- one less rule
- one less arbitrary value

Cons: 
- `10vh` will actually adapt to the viewport height
- `10vh !== .mt-5` (slightly bigger)

Reply:
We could, doesn't change that much. Let's go with `.mt-5`